modules/ROOT/pages/configure-hardening.adoc

```diff
@@ -0,0 +1,23 @@
+= Configure Environment Hardening on Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By design, Runtime Fabric automatically provides multiple aspects of environment hardening for your Mule installation. 
+For example, by leveraging Docker and Kubernetes, app isolation is achieved by default.  
+
+The following hardening checklist provides additional information about environment hardening already provided by 
```

Comment:
unclear on what "environment hardening" means. Sub with "Mule hardening"

Reply:
Got it - thanks!